queries.sql

```diff
@@ -68,4 +68,45 @@ SELECT neutered, AVG(escape_attempts) AS escapes FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg) AS minimum_weight, MAX(weight_kg) AS maximum_weight FROM animals GROUP BY species;
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
-SELECT species, AVG(escape_attempts) FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
+
+
+-- Write queries (using JOIN) to answer the following questions:
+-- Query to find the animals owned by Melody Pond
+SELECT owners.full_name, animals.name FROM owners JOIN animals ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+--  Query to list all the animals that are pokemon
+
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE animals.species_id = 1;
```

Comment:
- [ ] Kindly correct the data.sql before creating this query, you need to classify the species correctly otherwise you will have the Digimon 🤯  species as result.